compiler/rustc_target/src/spec/x86_64_apple_tvos.rs

```diff
@@ -1,12 +1,13 @@
-use super::apple_base::{opts, Arch};
+use super::apple_base::{opts, tvos_sim_llvm_target, Arch};
 use crate::spec::{StackProbeType, Target, TargetOptions};
 
 pub fn target() -> Target {
     let arch = Arch::X86_64_sim;
     Target {
-        llvm_target: "x86_64-apple-tvos".into(),
+        llvm_target: tvos_sim_llvm_target(arch).into(),
         pointer_width: 64,
-        data_layout: "e-m:o-i64:64-f80:128-n8:16:32:64-S128".into(),
+        data_layout: "e-m:o-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128"
```

Comment:
It seems all of our `x86_64-apple-*-*` have the same ABI string, should we try to be more proactive about bitrot by functionalizing or defaulting this? Or do we expect them to differ in the future?

Reply:
Not sure. In general this comes from LLVM/Clang via the magic of TableGen (I think), so I'm hesitant on trying to unify it given there's no reason for us to assume that these must be the same in the future.

I'll ask around though. I don't think we should try to unify these in this PR anyway.

Reply:
Leaving it for another PR sounds fine.